doc/lua_api.txt

```diff
@@ -1327,6 +1327,7 @@ Displays text on the HUD.
   text. Specify `0xFFFFFF` for white text, `0xFF0000` for red, and so on.
 * `alignment`: The alignment of the text.
 * `offset`: offset in pixels from position.
+* `size`: size of the text. The player-set font size is multiplied by size.x (y value isn't used).
```

Comment:
Why is this a table, if there's only one value in it? :thinking:

Reply:
It's used as a table by other element types

Reply:
Line too long. Please break it.

Reply:
Is a newline and two spaces the norm for doing that? Done if so

Reply:
Yes exactly. Indent to the first character after the list point.